api/nodes/views.py

```diff
@@ -243,6 +273,25 @@ def perform_create(self, serializer):
         user = self.request.user
         serializer.save(creator=user)
 
+    # Overrides BulkDestroyJSONAPIView
+    def bulk_destroy(self, request, *args, **kwargs):
+        """
+        Handles bulk destroy of nodes. Handles validation and enforces bulk limit.
+        """
+        kwargs['model'] = Node
```

Comment:
Every subclass will need to do this, which is prone to developer error and can be made more straightforward. Try to follow the open/closed (http://www.oodesign.com/open-close-principle.html) principle and expose a class variable (e.g. `model_class` and/or hook `get_model_class` instead.